### DISCUSSION TITLE
Figma components

### DISCUSSION THREAD

**Discussion by VHBS** [ekmas/neobrutalism-components#34] [Q&A] [answered]
It's possible share with us the figma design components?

**Comment by ekmas** [comment]
Sorry, but I don't have a figma file for this components, maybe I will add it in future.

Until then you can check the shadcn figma file: 
https://www.figma.com/community/file/1333848637032170061/shadcn-ui-design-system-2024
and basically update the styling and that's it.

**Comment by ekmas** [comment, marked as answer]
@VHBS There it is: [figma file](https://www.figma.com/community/file/1445024004618320019)
